projects/aca-content/src/lib/components/sidenav/navigation-menu/navigation-menu.component.html

```diff
@@ -0,0 +1,35 @@
+<button mat-icon-button [matMenuTriggerFor]="menu" aria-label="matMenu" class="app-menu-button">
+    <mat-icon svgIcon="menu" title="{{'APP.TOOLTIPS.OPTIONS_SETTINGS' | translate}}"></mat-icon>
+</button>
+<mat-menu #menu="matMenu" xPosition="before">
+    <div mat-menu-item class="app-menu-user-details" [routerLink]="['/profile']" title="{{'APP.TOOLTIPS.MY_PROFILE' | translate}}">
+        <button class="app-menu-user-details-button" aria-label="matMenu">
+            <mat-icon svgIcon="avatar"></mat-icon>
```

Comment:
this is incorrect,  according to existing ADF implementation this should be initials of the user, for example 'SA' for Super Admin, 'A' for admin, and so on... it's not the 'A'  icon for every user